src/main/java/BinaryTree/BinaryTreeTest.java

```diff
@@ -0,0 +1,38 @@
+package BinaryTree;
+
+public class BinaryTreeTest {
+
+    public static void main(String[] args) {
+
+        /*
+         *  Binary Tree
+         *  Tree data structure with at most 2 children per a parent node
+         *  Traverse through the tree using getLeft/RightNode();
+         *  Insert child node using setLeft/RightNode()
```

Comment:
Add details about 
- how it is not sorted which causes O(n) for search
- bounds on height